test/test_transformers.py

```diff
@@ -1412,7 +1412,7 @@ def test_flash_backward_sm86_headdim128(self):
         device = 'cuda'
         dtype = torch.float16
         make_tensor = partial(self.rand_tensor, type="dense", device=device, dtype=dtype)
-        # See check_gpu_sm86_head_dim_128 in pytorch/aten/src/ATen/native/transformers/cuda/sdp_utils.h
+        # See check_requires_grad_and_head_dim_gt64_and_sm_gt80 in pytorch/aten/src/ATen/native/transformers/cuda/sdp_utils.h
```

Comment:
I know, it's `check_requires_grad_and_head_dim_gt64_and_sm_gt86` not `check_requires_grad_and_head_dim_gt64_and_sm_gt80`